tests/manage/z_cluster/test_osd_heap_profile.py

```diff
@@ -0,0 +1,74 @@
+import logging
+import pytest
+import time
+import random
+
+from ocs_ci.framework.testlib import ManageTest, tier2, skipif_ocs_version, bugzilla
+from ocs_ci.ocs.resources.pod import get_ceph_tools_pod, get_osd_pods, get_osd_pod_id
+
+log = logging.getLogger(__name__)
+
+
```

Comment:
This test may not be relevant for external mode. Please add marker to skip it on external mode.